services/dolly/go-dolly

```diff
@@ -0,0 +1,17 @@
+#!/bin/ash
+
+timestamp=`date +%Y/%m/%d-%H:%M:%S`
+echo "go-dolly: I was called at $timestamp"
+
+config_path="/config/rclone/rclone.conf"
+data_path="/data/test"
+remote_name="remotebox"
+remote_path="isoblue_rclone"
```

Comment:
This should be setable in docker-compose

Reply:
Is there an clean in bash to make these values default but overwritable with envrion vars

Reply:
https://stackoverflow.com/a/2013589

or if live like @awlayton 

https://stackoverflow.com/a/28085062

I would argue that:

- `config_path` should be fixed and internal ... unless we will provide a way to completely override the auto generation of the  config
- `data_path` should be fixed and internal (your job is to put data there, see earlier comment on volume mounting) 
- `remote_name` is entirely internal I believe (?)
- `remote_path` is required input and dolly should fail if not provided (dolly should not just put data in your cloud storage where it wants)